modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Build.tsx

```diff
@@ -29,13 +29,28 @@ import {
 	TestrayCaseResult,
 	testrayCaseResultImpl,
 } from '../../../../services/rest';
-import {searchUtil} from '../../../../util/search';
+import {SearchBuilder} from '../../../../util/search';
 import useBuildTestActions from './useBuildTestActions';
 
+interface OutletContext {
+	runId: number | undefined;
+}
+
 const Build = () => {
 	const {buildId} = useParams();
 	const {updateItemFromList} = useMutate();
 	const {actions, form} = useBuildTestActions();
+	const {runId} = useOutletContext<OutletContext>();
+
+	const caseResultFilter = new SearchBuilder();
+
+	const filtersTest = runId
```

Comment:
"filter test" is that so ?  or did you forget to change ?

```suggestion
	const filtersTest = runId
```

Reply:
:man_facepalming: Sorry I really forgot to change this, thanks. :handshake: